src/test/jtx/impl/flags.cpp

```diff
@@ -51,7 +51,9 @@ void
 nflags::operator()(Env& env) const
 {
     auto const sle = env.le(account_);
-    if (sle->isFieldPresent(sfFlags))
+    if (!sle)
```

Comment:
Before line 44 (above), for the sake of consistency, we need to introduce a similar check

Reply:
Good catch. Fixed.